src/utils/index.ts

```diff
@@ -0,0 +1,9 @@
+import * as obj from './object';
+import * as arr from './array';
+
+const tutureUtilities = {
+  ...arr,
+  ...obj,
+};
+
+export default tutureUtilities;
```

Comment:
I still think these utilities a little bit funny😂Maybe you can take a look at this [package](https://www.npmjs.com/package/isarray). Also, I'm in favor of dropping support for older browsers, since Tuture is designed for developers  and Chrome should be their primary weapon.

Reply:
Although, you may think `isArray` is too simple, But It just one of our utilities functions. 

eg: 

`isObjectEmpty` need more than one line code. And many place need it, So we need write it to our `tutureUtilities` packages.

```javascript
const isObjectEmpty = (obj: any): boolean => {
  const keyArray = Object.keys(obj);
  return keyArray.length === 0;
};
```

Reply:
My destination is to create a common-use javascript utilities packages. 🤠